Frame.cc

```diff
@@ -1252,15 +1261,15 @@ bool Frame::GetPointSpectralData(std::vector<float>& data, int region_id, casaco
                 // start the timer
                 auto t_start = std::chrono::high_resolution_clock::now();
                 // check if region point changed from subimage point
-                if ((region_id == CURSOR_REGION_ID) && (Interrupt(_cursor_xy, subimage_cursor))) {
+                if ((region_id == CURSOR_REGION_ID) && (Interrupt(region_id, _cursor_xy, subimage_cursor))) {
                     return false; // cursor moved
```

Comment:
Return values for GetPointSpectralData and GetRegionSpectralData are not checked, so if the profile is interrupted it is still sent to the frontend.  I think this profile is no longer needed.